src/guide/index.md

```diff
@@ -6,34 +6,37 @@ order: 2
 
-Machine Learning is a field that seems rather exclusive to people with academic background (professors, Phd’s, …). It’s often not trivial to follow even beginner guides since the field comes with a heavy focus on mathematical theories. On top of that, setting things up to just get started is also very challenging, especially if one isn’t a data scientist who is familiar with all required tools.
-
-We try to lower the barrier of entry as well as making the tooling scape more pleasant, so more people are able to start learning in this field, by implementing the following features:
+MachineLabs tries to lower the barrier of entry as well as making the tooling scape more pleasant so more people are able to start learning in this field. This includes: 
 
 * **Online Code Editor** - Everyone should be able to simply open up a new browser tab and get started. No setup hassle, no prerequisites required. It’s possible to literally copy and paste Machine Learning code and execute it right in the browser.
 
-* **Ready to use environments for main ML frameworks** - Apart from Tensorflow and Theano, there’s other major libraries like PyTorch or Caffe. MachineLabs provides execution environments for all of these including common libraries needed to perform Machine Learning tasks. Setting up and using a particular environment is as simple as changing one line of configuration code. 
+* **Ready to use environments for main ML frameworks** - Apart from **Tensorflow** and **Theano**, there’s other major libraries like **PyTorch** or **Caffe**. MachineLabs provides execution environments for all of these including common libraries needed to perform Machine Learning tasks. Setting up and using a particular environment is as simple as changing one line of configuration code. 
 
-* **Access to blazingly fast GPU hardware** - Training a neural net can take hours, or even several days to finish. That’s why we want to execute our code on super fast GPU optimized machines. MachineLabs offers GPU accelerated machines as part of an advanced pricing plan.
+* **Access to blazingly fast GPU hardware** - Training a neural net can take hours, or even several days to finish. That’s why data scientists want to execute their code on super fast GPU optimized machines. MachineLabs offers GPU accelerated machines as part of an advanced pricing plan.
 
-* **Keep track of work** - All executions performed stay available for us so we can always keep track of our experiments. Each execution offers information about the duration of training, environment and execution status.
+* **Keep track of work** - All executions performed stay available so users can always keep track of their experiments. Each execution offers information about the duration of training, environment and execution status.
```

Comment:
You rather want to speak *about* the users here? I thought it's good to speak from the users perspective. Maybe even speak directly to him

>All executions performed stay available for you..

Reply:
The reason I did this is because the introduction part introduces MachineLabs as a product. So it explains what Machinelabs does and what users can do with it. Unlike actual guides where we're sitting together with a user in the same boat and learn how to do certain things.

Reply:
That's in your head but does the user feel the same about it? After all, they are reading an introduction about the platform. Personally, I think if we are speaking about the same audience that reads it, we should go with a more inclusive perspective.

I would only speak *about* the user if the reader audience wouldn't be the users itself. E.g. if this was a document for investors, it would be ok to speak *about* the users. 

I don't have super strong feelings about this, if you insist on these changes. Just my general feeling.

//cc @d3lm

Reply:
To move on here, I went with "you" now